includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -1795,13 +1796,19 @@ public function add_status_change_note( $order, $charge_permission_id, $new_stat
 	 * @param string $id The value to provide the id attribute of the script with.
 	 * @return void
 	 */
-	public function update_js( $id = 'wc-apa-update-vals' ) {
+	public function update_js( $id = '' ) {
+
+		if ( empty( $id ) ) {
```

Comment:
If coding style allows, can we use `$id = $id ?: 'wc-apa-update-vals'` instead?

Reply:
Short ternary produces a phpcs, can do a full ternary though.

> Using short ternaries is not allowed
> (WordPress.PHP.DisallowShortTernary.Found)

Reply:
Gotcha, thanks for giving it a try!

Reply:
Thanks!